src/olympia/landfill/management/commands/generate_ui_test_addons.py

```diff
@@ -83,5 +91,3 @@ def handle(self, *args, **kwargs):
             for addon in hero_addons:
                 serializer.create_named_addon_with_author(addon)
             serializer.create_installable_addon()
-        cache.clear()
-        call_command('clear_cache')
```

Comment:
This is really old code that's pre cache-machine removal and still from when this was behind an API, which it isn't anymore.

The whole thing could be simplified and not written as a serializer but that's another cleanup.